packages/thirdweb/src/wallets/in-app/core/wallet/types.ts

```diff
@@ -69,6 +69,7 @@ export type InAppWalletCreationOptions =
        * Metadata to display in the Connect Modal
        */
       metadata?: {
+        name?: string;
```

Comment:
Nice! 

This is actually useful for our own connect modal too.

Could replace the "social login" default name that we show in the wide modal.

As for the icon, can you actually add a separate prop for it instead of overloading image?

So name / icon / image.

That way we can also use it in the connect modal to let you replace the icon separately to the image that shows above the sign in options.

Reply:
did it for you :) merging shortly